Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
```

Comment:
@grdryn indeed `RUN some command` is the shell form. You should be able to combine it all with `&&`?
And you  can still have comments in between the lines :)